packages/pigeon/platform_tests/test_plugin/ios/Classes/CoreTests.gen.swift

```diff
@@ -652,6 +652,66 @@ class CoreTestsPigeonCodec: FlutterStandardMessageCodec, @unchecked Sendable {
   static let shared = CoreTestsPigeonCodec(readerWriter: CoreTestsPigeonCodecReaderWriter())
 }
 
+private class PigeonStreamHandler<T>: NSObject, FlutterStreamHandler {
```

Comment:
super optional nit: can also use a multiple letters for generic type (e.g. `PigeonStreamHandler<Event>`)